ice/classes/sanger_analysis.py

```diff
@@ -669,7 +678,7 @@ def analyze_and_rank(self):
 
         # create totals:
 
-        aggregated_indel = {}
+        aggregated_indel = defaultdict(float)
```

Comment:
Should be able to delete the if statement on line 685 since you changed to defaultdict